ember-toucan-core/src/components/form/textarea-field.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="flex flex-col">
+  <Form::Field as |field|>
+    <field.Label for={{field.id}} data-textarea-label>{{@label}}</field.Label>
```

Comment:
Excellent points.  Yeah this could get tricky.  I wasn't quite sure either.  I think it's important for us to splat the attributes onto the underlying control so that folks can add modifiers, attributes, etc., but then that makes access child elements of the Field a lot trickier.

So are you thinking something like:

```
<div class="flex flex-col" data-test-selector={{@testSelector}}>
   <Form::Field as |field|>
     <field.Label for={{field.id}} data-label>{{@label}}</field.Label>
```

As you say, then we could allow for selectors like `[data-test-selector="Simon"] [data-label]`.  Maybe we should provide a fractal-page-object to help with some of this?

Reply:
yeah, this could work.

> Maybe we should provide a fractal-page-object to help with some of this?

Hm, maybe? I haven't yet touched fractal page objects since joining CS, so can't really speak to it. Are we providing these in other OSS addons, or are they very specific to CS?

Reply:
This is an example of the test helpers provided in ember-headless-table: https://ember-headless-table.pages.dev/docs/testing